src/utils.ts

```diff
@@ -1,21 +1,47 @@
 import { Dispatch } from "./dispatch";
 
 export class Util {
-    async getAllPages(path: string, arrayName: string, dispatch: Dispatch) {
+    static async getAllPages(path: string, arrayName: string, dispatch: Dispatch, filter?: string) {

```

Comment:
Unify filter/paging. Before it had an issue that filter + paging did not work together.